CMakeLists.txt

```diff
@@ -296,7 +296,9 @@ endif (UNIX AND NOT ANDROID)
 # link android logging library
 if (ANDROID)
     find_library (log-lib log)
+    target_link_libraries (${ML_MODULE_NAME} PRIVATE log)
     target_link_libraries (${BOARD_CONTROLLER_NAME} PRIVATE log)
+    target_link_libraries (${DATA_HANDLER_NAME} PRIVATE ${DSPFILTERS} ${WAVELIB} log)
```

Comment:
no need to link dspfilter and wavelib here, in cmake target_link_libraries works incrementally